src/widgets/Table.ts

```diff
@@ -20,7 +20,7 @@ export const Table = ConfiguredWidget<TableData, TableConfig>({
     const columnWidths = headers.map((header, index) => {
         let width = Math.max(
             header.length, 
-            rows.reduce((maxWidth, row) => Math.max(maxWidth, row[index].toString().length), 0), 
+            rows.reduce((maxWidth, row) => Math.max(maxWidth, (row[index] || 'undefined').toString().length), 0), 

```

Comment:
Prefer `??` to `||`.

Does it make sense to use an empty string instead of `'undefined'` as the default value?

Reply:
That makes sense, but it might be confusing for people to debug

Reply:
That's reasonable. Is it worth distinguishing between `null` and `undefined`?

Reply:
Probably not. But I never use null purposefully so I don't have the best judgement

Reply:
We could just say 'no value'

Reply:
After thinking this over, let's take a slightly different approach. This just sets the width of the column; we'll have to set the content separately.

Let's do this instead:

```
let rows = data
  .slice(0, height - labelHeight)
  .map(row => row.map(cell => cell === null ? 'null' : cell === undefined ? 'undefined' : cell.toString()))
```

This will map all values to a corresponding string, and the width check should just work.